tedium.ts

```diff
@@ -64,15 +64,14 @@ const cli = cliArgs([
       }
       throw new Error(`invalid max changes, expected an integer: ${x}`);
     },
-    alias: 'c',
-    description: 'The maximum number of repos to push. Default: 0',
-    defaultValue: 0
+    description: 'The maximum number of repos to push. Default: 1',
+    defaultValue: 1
```

Comment:
This is less conservative, isn't it? By default previously it wouldn't push anything, so you would first run it without `-c` and you could eyeball the changes it would make